packages/mobile/src/components/lineup-tile/CollectionTile.tsx

```diff
@@ -0,0 +1,229 @@
+import { useCallback, useMemo } from 'react'
+
+import {
+  FavoriteSource,
+  PlaybackSource,
+  RepostSource,
+  ShareSource
+} from 'audius-client/src/common/models/Analytics'
+import { Collection } from 'audius-client/src/common/models/Collection'
+import { Track } from 'audius-client/src/common/models/Track'
+import { User } from 'audius-client/src/common/models/User'
+import { getUserId } from 'audius-client/src/common/store/account/selectors'
+import {
+  EnhancedCollectionTrack,
+  getCollection,
+  getTracksFromCollection
+} from 'audius-client/src/common/store/cache/collections/selectors'
+import { getUserFromCollection } from 'audius-client/src/common/store/cache/users/selectors'
+import {
+  repostCollection,
+  saveCollection,
+  undoRepostCollection,
+  unsaveCollection
+} from 'audius-client/src/common/store/social/collections/actions'
+import {
+  OverflowAction,
+  OverflowSource
+} from 'audius-client/src/common/store/ui/mobile-overflow-menu/types'
+import { requestOpen as requestOpenShareModal } from 'audius-client/src/common/store/ui/share-modal/slice'
+import { albumPage, playlistPage } from 'audius-client/src/utils/route'
+import { open as openOverflowMenu } from 'common/store/ui/mobile-overflow-menu/slice'
+import { isEqual } from 'lodash'
+
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+import { CollectionTileTrackList } from './CollectionTileTrackList'
+import { LineupTile } from './LineupTile'
+import { LineupItemProps } from './types'
+
+export const CollectionTile = (props: LineupItemProps) => {
+  const { uid } = props
+
+  const collection: Collection = useSelectorWeb(
+    state => getCollection(state, { uid }),
+    isEqual
+  )
+
+  const tracks: EnhancedCollectionTrack[] = useSelectorWeb(
+    state => getTracksFromCollection(state, { uid }),
+    isEqual
+  )
+
+  const user: User = useSelectorWeb(
+    state => getUserFromCollection(state, { uid }),
+    isEqual
+  )
+
+  if (!collection || !tracks || !user) {
+    console.warn(
+      'Collection, tracks, or user missing for CollectionTile, preventing render'
+    )
+    return null
+  }
+
+  if (collection.is_delete || user?.is_deactivated) {
+    return null
+  }
+
+  return (
+    <CollectionTileComponent
+      {...props}
+      collection={collection}
+      tracks={tracks}
+      user={user}
+    />
+  )
+}
+
+const CollectionTileComponent = ({
+  collection,
+  togglePlay,
+  tracks,
+  user,
+  ...props
+}: LineupItemProps & {
```

Comment:
might be nice to categorize this as a type, just to potentially export it if needed?